vars/main.yml

```diff
@@ -3,20 +3,23 @@
 
 dependencies_deb:
   - python-dev
+  - python-pip
```

Comment:
Is monasca agent installed in a virtualenv? Would this be required in that case?

Reply:
It is - I could probably use pip in there to install the stackhpc-monasca-agent-plugins repo from git. That would save some redundancy.

Reply:
Perhaps I'm misunderstanding, but don't you have to use the pip in the virtualenv?

Reply:
Not wanting to bikeshed too much, if this is required then feel free to merge :)

Reply:
Pip is required here to install plugins from a git repo to /var/lib/monasca/.. which is the default path checked by the monasca-agent. This path is outside of the monasca-venv. It would be nice if it used entry points, but all it does is check the folder for *.py files.